package.json

```diff
@@ -19,9 +19,9 @@
   },
   "homepage": "https://github.com/ForestAdmin/forest-rails#readme",
   "devDependencies": {
-    "@commitlint/cli": "8.3.5",
-    "@commitlint/config-conventional": "8.3.4",
-    "@commitlint/travis-cli": "8.3.5",
+    "@commitlint/cli": "9.1.1",
+    "@commitlint/config-conventional": "9.1.1",
+    "@commitlint/travis-cli": "9.1.1",
```

Comment:
See the "light" breaking change introduced by version 9:
https://github.com/conventional-changelog/commitlint/blob/master/CHANGELOG.md#breaking-changes